.drone.star

```diff
@@ -1844,7 +1845,8 @@ def build():
             "name": "build",
             "image": OC_CI_GOLANG,
             "commands": [
-                "make -C ocis build",
+                "curl -SsfL -o /usr/local/bin/retry https://raw.githubusercontent.com/owncloud-ci/retry/master/retry && chmod +x /usr/local/bin/retry",
```

Comment:
golang images have "retry" in them now - https://github.com/owncloud-ci/golang/pull/91
So this line should not be needed.